src/DynamoCoreWpf/Views/GuidedTour/RealTimeInfoWindow.xaml

```diff
@@ -0,0 +1,88 @@
+﻿<Popup x:Class="Dynamo.Wpf.Views.GuidedTour.RealTimeInfoWindow"
+       xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+       xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+       xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+       xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+       xmlns:local="clr-namespace:Dynamo.Wpf.Views.GuidedTour"
+       xmlns:ui="clr-namespace:Dynamo.UI"
+       xmlns:p="clr-namespace:Dynamo.Wpf.Properties;assembly=DynamoCoreWpf" 
+       mc:Ignorable="d" 
+       AllowsTransparency="True"
+       Opacity="0.5"
+       Width="{Binding Width}"
+       MinWidth="340"
+       Height="{Binding Height}"
+       d:DesignHeight="67" 
+       d:DesignWidth="340">
+    <Popup.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ui:SharedResourceDictionary Source="{x:Static ui:SharedDictionaryManager.DynamoConvertersDictionaryUri}" />
+                <ui:SharedResourceDictionary Source="{x:Static ui:SharedDictionaryManager.DynamoModernDictionaryUri}" />
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </Popup.Resources>
+    <Border BorderBrush="{StaticResource RealTimeInfoWindowBackgroundColor}" 
+            Background="{StaticResource RealTimeInfoWindowBackgroundColor}"
+            BorderThickness="1" 
+            CornerRadius="2">
+        <Grid Background="Transparent">
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="10"/>
+                <ColumnDefinition Width="Auto"/>
+                <ColumnDefinition Width="Auto"/>
+                <ColumnDefinition Width="*"/>
+            </Grid.ColumnDefinitions>
+            <Line X1="1" 
```

Comment:
What are the lines here for? Some comments would be good

Reply:
@QilongTang I added some comments in this xaml, please let me know if they should be more descriptive.
commit: https://github.com/DynamoDS/Dynamo/pull/11918/commits/cc26a3d36d1f186cbc0d598fc4e1c1d3f1d42b31
Thanks